homeassistant/components/media_player/webostv.py

```diff
@@ -268,11 +259,38 @@ def media_content_type(self):
     @property
     def media_title(self):
         """Title of current playing media."""
-        if (self._channel is not None) and ('channelName' in self._channel):
-            return self._channel['channelName']
-        else:
+        return self._title
+
+    @property
+    def media_channel(self):
+        """Channel currently playing."""
+        return self._channel
+
+    @property
+    def media_position(self):
+        """Position of current playing media in seconds."""
+        if self._progress is None:
             return None
 
+        position = self._progress
+
+        if self._state == STATE_PLAYING:
+            position += (util.dt.utcnow() -
```

Comment:
No, this is not needed. This property returns only the stored progress pointer. The frontend will do this based on the `media_position_updated_at`.  This is to avoid unnecessary state updates.